File: pytools/data/data.code.php.server/ReqSceneEnter.php
<?php
namespace protocol;

use network\Packet;


class ReqSceneEnter
{
	private $doorId = 0;


	public function __construct(&$packet)
	{
		$this->decode($packet);
	}

	public function decode(&$packet)
	{
		$this->doorId = $packet->readU32();
	}


	public function getDoorId()
	{
		return $this->doorId;
	}

}

File: pytools/examples/java/server/protocol/ReqSceneEnter.java
package protocol;


import network.Packet;
import network.PacketUtil;

import java.util.List;
import java.util.ArrayList;


public class ReqSceneEnter
{
	private int door_id;


	public ReqSceneEnter(Packet packet)
	{
		door_id = packet.readInt();
	}



	public long getDoorId()
	{
		return PacketUtil.readUInt(this.door_id);
	}

}
